src/main/java/services/ConnectionConfig.java

```diff
@@ -0,0 +1,30 @@
+package services;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionConfig {
```

Comment:
this class is not a service. move it to:
1. `repository` package
2. or to separate packge for configurations: `configuration`